### PR TITLE
Consul readme

### PR DESCRIPTION
**Pull request by lwander**
@duftler 

Here I'll be updating the consul + spinnaker integration documentation w.r.t. consul setup. I'll add documentation as the scripts for setup are added. I think this leaves a nice audit trail.

### CODE REVIEW

**Comment by lwander** [comment]
ping

File: consul/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+> _This is a work in progress, and not complete yet. Please hold off on using
+> this guide until this message disappears_
+
+There are two ways to use Consul + Spinnaker. The first assumes you don't have
+Consul running in an existing deployment, and the second assumes you do. If you
+fall in the first bucket, continue reading.
+
+## Configuring Consul From Scratch
+
+This assumes you're not using Consul for service discovery.
```
Comment (duftler) [inline consul/README.md]: s/you're not/you're not already

File: consul/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+### Installing Consul
+
+If you're using [Consul](consul.io), Spinnaker assumes that you've taken care of 
+installing the Consul agent on each machine you want to be discoverable. The 
+best way to take care of this is to create a base image with Consul installed,
+and install further packages on top of that. To do so, copy the file in
+`./install/install.sh` onto a VM, run `$ sudo install.sh`, and capture the
+resulting disk as a VM. There's many ways to do this (Packer,
```
Comment (duftler) [inline consul/README.md]: s/There's/There are

**Comment by duftler** [comment]
LGTM after addressing comments.

**Pull request merged**